R/impact_central.R

```diff
@@ -202,3 +196,106 @@ determine_vaccine_delivery <- function(meta1){
 #
 #
 # }
+
+#' Calculate impact by year of vaccination country perspective
+#'
+#' This will calculate the impact by year of vaccination by country and
+#' burden outcome for a single disease and vaccine.
```

Comment:
Is it useful to write up the method here?

Reply:
excitingly we reverted the name to "activity perspective" but I think having a sentence on the method may be helpful